rc/filetype/janet.kak

```diff
@@ -19,7 +19,7 @@ hook global WinSetOption filetype=janet %{
     hook -once -always window WinSetOption filetype=.* %{ remove-hooks window janet-.+ }
 }
 
-hook -group janet-highlight global WinSetOption filetype=janet %{
+hook -group janet global WinSetOption filetype=janet %{
```

Comment:
I think this should keep the `janet-highlight` group, for consistency with other filetypes